packages/astro-purgecss/src/index.ts

```diff
@@ -152,6 +152,11 @@ function Plugin(options: PurgeCSSOptions = {}): AstroIntegration {
                 return join(outDir, 'index.html');
               }
 
+              // Handle custom 404 page (https://docs.astro.build/en/basics/astro-pages/#custom-404-error-page)
+              if (pathname === '404/') {
```

Comment:
Hey @Jaanilj the `/` is required here ? I was thinking of using includes instead ? something like:

```
if (pathname.includes('404')) {
}
```

Reply:
True! That seems a bit more safe / simple! I'll update 👍

Reply:
Actually, I'm not 100% sure. Not sure if that could cause problems with nested `404` pages? For example I think it's not uncommon to have `/en/404.html`, `/dz/404.html`, etc. for language localised `404` error pages.

If we use `pathname.includes('404')` it might break the generation for those non-root `404` pages. This is the special case for "catch-all" root level 404 page.

WDYT?

Reply:
Hmm, I see what you mean, then let's only handle root case. for now.

Reply:
Yeah. I can open a new PR if I find issues with other types of "special" Astro pages 👍